### DISCUSSION TITLE
Adding Ads with plausible, GPRD compliant?

### DISCUSSION THREAD

**Discussion by NickG13** [plausible/analytics#922] [Anything Else]
Hi, 
I just am thinking on paying for Plausible, but I'm also thinking on adding google adsense on my site. Will Plausible + Google Adsense will be GPDR compliant without a need of a cookie banner? If not, which other addons can help me implement ads without the need of banners? 

Thank you !

**Comment by ukutaht** [comment]
Unfortunately Google Adsense does place a cookie and requires consent: https://support.google.com/adsense/answer/1348695?hl=en

You can check some alternative advertising networks like these:
https://www.ethicalads.io/
https://intravert.co/marketplace/
https://www.carbonads.net/

I think these are mostly limited to developer audiences, though.

**Reply by NickG13** [reply]
Thank you ukutaht, I may not be showing ads then. I may be adding affiliate links to monetice then, thanks!
